### PR TITLE
ntfs-3g: new packages at 2017.3.23

### PR DESCRIPTION
**Pull request by Bridouz**
## Description of package

  - NTFS filesystem driver and utilities to access ntfs filesystems

## Checklist

- [X] Latest upstream version.
- [X] I agree to maintain this package (**required**).
- [X] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/ntfs-3g/build [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+./configure --prefix=/usr
+
+make
+make DESTDIR="$1" install
+
+# simlink for uniform behaviour
+ln -s ntfs-3g "$1/sbin/mount.ntfs" 
+
+#remove docs
+rm -rf "$1"/usr/share/doc/ln
```
Comment (dylanaraps) [inline community/ntfs-3g/build]: This can be removed (see `KISS_RM` in `man kiss`).

File: community/ntfs-3g/build [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
+
+make
+make DESTDIR="$1" install
+
+# simlink for uniform behaviour
+ln -s ntfs-3g "$1/sbin/mount.ntfs"
```
Comment (dylanaraps) [inline community/ntfs-3g/build]: Comment can be removed. Also: `ln -s ntfs-3g "$1/usr/bin/mount.ntfs`

File: community/ntfs-3g/build [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
```
Comment (dylanaraps) [inline community/ntfs-3g/build]:
```sh
./configure \
    --prefix=/usr
```

**Comment by dylanaraps** [comment]
There's a weird binary file as a part of this PR.

**Comment by Bridouz** [comment]
Everything is ok I think.

**Pull request merged**